### ISSUE TITLE
Thunderbird compatibility?

### ISSUE THREAD

**Issue by emansom** [rafaelmardojai/firefox-gnome-theme#417]
Almost a decade ago now, there [used to be](https://github.com/gnome-integration-team/thunderbird-gnome) a Adwaita theme for Thunderbird. Given Thunderbird leverages much of the same stack that Firefox does, I presume forking this codebase for compatibility with Thunderbird is a possibility.

Is that a correct assumption, and are you welcoming any pull requests for this, eventual project overhaul to split into `firefox` and `thunderbird` variants that inherit from the same `base`?

**Comment by rafaelmardojai** [comment]
Yeah, I guess it could work. I haven't inspected Thunderbird UI, but it's posible that some code can be shared.

The only problem I see with Thunderbird theming is that the the toolbar changes drastically depending on the tab, so you would have to find a GNOME layout that works with that. Maybe turn the tabs into a ViewSwitcher.

An alternative could be to wait for the [announced redesign](https://www.omglinux.com/major-thunderbird-redesign-early-look/) and start from there.

**Comment by ArtyIF** [comment]
Yeah, I can see how the redesign can be converted. It's kind of like Geary

**Comment by emansom** [comment]
Gonna be a while. Will close for now and check back when it lands in Thunderbird beta branch.

https://twitter.com/mozthunderbird/status/1546818615453827076

**Comment by xplosionmind** [comment]
The redesign is coming up… do you think it makes sense to create a different repository linked to this one, or simply Thunderbird-specific customizations can be added to this repository?

**Comment by emansom** [comment]
The new redesign [has been released](https://www.phoronix.com/news/Thunderbird-115).

**Comment by rafaelmardojai** [comment]
I don't know if I should create some alt project named `gecko-libadwaita-css` or something and build there a shared sass or less library that both themes can use.

**Comment by rafaelmardojai** [comment]
I will start making a straight port of the theme to thunderbird and then see what's the best way to share code.

Started playing with the headerbar:

![Captura desde 2023-07-11 18-59-59-obfuscated](https://github.com/rafaelmardojai/firefox-gnome-theme/assets/6210397/5a64ab47-068d-4c40-bc01-61dd3567b5e8)

![Captura desde 2023-07-11 19-00-27-obfuscated](https://github.com/rafaelmardojai/firefox-gnome-theme/assets/6210397/15e0e3dd-5783-4611-aa2c-568da8adc3d5)

**Comment by rafaelmardojai** [comment]
Some progress:

![Captura desde 2023-07-12 13-15-00-obfuscated](https://github.com/rafaelmardojai/firefox-gnome-theme/assets/6210397/efa0aa1f-485a-4abd-a43f-767bb11d70bf)

**Comment by xplosionmind** [comment]
So cool! You are the best @rafaelmardojai

**Comment by rafaelmardojai** [comment]
If anyone want's to try out, the code is here: https://github.com/rafaelmardojai/thunderbird-gnome-theme. Issues are closed for now since it's work in progress.

**Comment by emansom** [comment]
> If anyone want's to try out, the code is here: https://github.com/rafaelmardojai/thunderbird-gnome-theme. Issues are closed for now since it's work in progress.

[Packaged it and published it](https://aur.archlinux.org/packages/thunderbird-gnome-theme-git) on the AUR for people to test. 👍🏻
